GitTfsTest/Core/GitChangeInfoTests.cs

```diff
@@ -245,5 +245,28 @@ public void MultipleChangesWithJapanese()
                 Assert.Equal("TestFiles/試し4.txt", changes[4].path);
             }
         }
+
+        [Fact]
+        public void NotRepresentableChanges()
```

Comment:
Could you cut this test in 2 (one for Add/Delete case and one for Renaming) with a naming explicit (I know that the other tests are not very well named :( )